edx/edx-enterprise
# ENT-218/ENT-221 Implementing clients for success factors endpoints

**Description:** Implemented the client functions needed to interact with the success factors api.
Also began implementation of the transmitters that will be called from the processes Jill and Jesse are building.

**JIRA:** Link to JIRA ticket

**Dependencies:** https://github.com/edx/edx-enterprise/pull/59/files
https://github.com/edx/edx-enterprise/pull/63/files

**Merge deadline:** List merge deadline (if any)

**Installation instructions:** List any non-trivial installation 
instructions.

**Testing instructions:**

See https://docs.google.com/document/d/13vol2OQT_vVzD164k7fh7TH2_xsGtj1rWb4xWC3ozAg/edit

**Merge checklist:**

  **DO NOT** just add dependencies to `requirements/*.txt` files.

**Post merge:**

**Author concerns:** List any concerns about this PR - inelegant 
solutions, hacks, quick-and-dirty implementations, concerns about 
migrations, etc.
